ext/openssl/xp_ssl.c

```diff
@@ -990,7 +990,10 @@ static char *php_openssl_cipher_get_version(const SSL_CIPHER *c, char *buffer, s
 {
```

Comment:
This function doesn't make any sense to me. It was added in https://github.com/php/php-src/commit/d8580cb286bbe2f4eb29616534882c73aa4882d7, but without an explanation for why it is needed. Maybe @bukka remembers?

Reply:
Okay, I found the reason. add_assoc_string used to accept char* instead of const char*. Now it accepts const char*, so I have removed this workaround in https://github.com/php/php-src/commit/626bc0e7cc9c0ab3054de371ae5bbf33de4883e7.

Reply:
yeah that was the reason

Reply:
I think :)